gh-actions/common/build-debian/action.yml

```diff
@@ -106,7 +106,12 @@ runs:
         artifacts-dir: ${{ env.BUILD_OUTPUT_DIR }}
         source-dir: ${{ env.BUILD_INPUT_DIR }}
         docker-image: ${{ inputs.docker-image }}
-    - name: Upload artifacts
+    - name: Copy source package to output dir
+      shell: bash
+      run: |
+        set -eu
+        cp -r ${{ env.SOURCE_OUTPUT_DIR }}/* ${{ env.BUILD_OUTPUT_DIR }}
```

Comment:
IMHO, we should do this phase just after building the source, and ensure that this happens also if binary build fails: This can help a lot to debug situations failure situations, because one can download the sources locally and build them with `sbuild` (or similar tools) to fix the inconsistencies.

Also in commit message, we mostly spell it as "art_i_fact" even if both words seems to be correct.